src/shared/components/chevron/Chevron.scss

```diff
@@ -1,5 +1,10 @@
 @import 'src/styles/common';
 
+.wrapper {
+  display: inline-block;
+  font-size: 0; // to prevent line-height's impact on the height of the element
+}
+
 .chevron {
   fill: var(--chevron-fill, $blue);
```

Comment:
shouldn't this be for `svg`?

Reply:
The chevron class in Chevron.scss is assigned to the svg. See Chevron.tsx and the review deployment:

![image](https://user-images.githubusercontent.com/6834224/182176376-ddde66fe-d728-4351-b848-1c20216fd5c4.png)